daprdocs/content/en/developing-applications/building-blocks/bindings/bindings-overview.md

```diff
@@ -6,56 +6,72 @@ weight: 100
 description: Overview of the bindings API building block 
 ---
 
-Using Dapr's bindings API, you can both trigger your app with events coming in from external systems and interface with external systems. In addition, the Dapr binding API enables you to:
+Using Dapr's bindings API, you can trigger your app with events coming in from external systems and interface with external systems. With the bindings API, you can:
 
-- Remove the complexities of connecting to and polling from messaging systems, such as queues and message buses
-- Focus on business logic, instead of the implementation details of how to interact with a system
+- Avoid the complexities of connecting to and polling from messaging systems, such as queues and message buses
```

Comment:
Should there be periods at the end of all these sentences? IMO it should to me it read easier.